database/elastic_mapping/mappings.ts

```diff
@@ -112,6 +112,12 @@ const relationshipType = () => ({
   label: text,
   value: id,
   type: noIndexText,
+  inheritedValue: {
+    properties: {
+      value: id,
```

Comment:
What happens if the inherited value is a plain-storing property like text or number or date? Should this mapping not be dependent upon inherited property?

Reply:
Indeed, we will need to put a mapping according to the property being inherited and update the mapping if that changes.

Reply:
oh! that was so obvious, i guess we were too focused on the multiselect/select combos :(

Reply:
aparently id mapping works for all the cases and the crashes where produced by other conditions.

Reply:
id mapping does not work for geolocation.